src/Network/Network/NetworkWatcher/NetworkConfigurationDiagnostic/NewAzureNetworkWatcherConfigurationDiagnosticProfileCommand.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkWatcherNetworkConfigurationDiagnosticProfile"), OutputType(typeof(PSNetworkConfigurationDiagnosticProfile))]
```

Comment:
@irrogozh this cmdlet is being flagged as not support `ShouldProcess` due to the `New` verb, but since it's only creating an in-memory object, this cmdlet does not need to support it. You can suppress the exception by adding the exception line to the corresponding `SignatureIssues.csv` file. More information can be found [here](https://github.com/Azure/azure-powershell/blob/master/documentation/Debugging-StaticAnalysis-Errors.md#signature-issues)

Reply:
Done